tools/deploy.ps1

```diff
@@ -0,0 +1,66 @@
+$ErrorActionPreference = "Stop"
+
+Push-Location (Get-Item $PSScriptRoot).Parent.FullName
+
+try {

```

Comment:
A moustache on the same line as a `try`? What is this, Java?

Reply:
Actually, strike that. We seem to have a lot of moustaches on same lines in our powershell. But later on you have

Reply:
Better now?

Reply:
I still see

Reply:
And now?